doc/release-notes.rst

```diff
@@ -2,6 +2,112 @@
+
+  * The original librados rados_objects_list_open (C) and objects_begin
+    (C++) object listing API, deprecated in Hammer, has finally been
+    removed.  Users of this interface must update their software to use
+    either the rados_nobjects_list_open (C) and nobjects_begin (C++) API or
+    the new rados_object_list_begin (C) and object_list_begin (C++) API
+    before updating the client-side librados library to Luminous.
+    Object enumeration (via any API) with the latest librados version
+    and pre-Hammer OSDs is no longer supported.  Note that no in-tree
+    Ceph services rely on object enumeration via the deprecated APIs, so
+    only external librados users might be affected.
+
+    The newest (and recommended) rados_object_list_begin (C) and
+    object_list_begin (C++) API is only usable on clusters with the
+    SORTBITWISE flag enabled (Jewel and later).  (Note that this flag is
+    required to be set before upgrading beyond Jewel.)
+
+- *CephFS*:
+
+  * When configuring ceph-fuse mounts in /etc/fstab, a new syntax is
+    available that uses "ceph.<arg>=<val>" in the options column, instead
+    of putting configuration in the device column.  The old style syntax
+    still works.  See the documentation page "Mount CephFS in your
+    file systems table" for details.
+
+  * CephFS clients without the 'p' flag in their authentication capability
+    string will no longer be able to set quotas or any layout fields.  This
+    flag previously only restricted modification of the pool and namespace
+    fields in layouts.
+  * CephFS directory fragmentation (large directory support) is enabled
+    by default on new filesystems.  To enable it on existing filesystems
+    use "ceph fs set <fs_name> allow_dirfrags".
+  * CephFS will generate a health warning if you have fewer standby daemons
+    than it thinks you wanted.  By default this will be 1 if you ever had
+    a standby, and 0 if you did not.  You can customize this using
+    ``ceph fs set <fs> standby_count_wanted <number>``.  Setting it
+    to zero will effectively disable the health check.
+  * The "ceph mds tell ..." command has been removed.  It is superceded
+    by "ceph tell mds.<id> ..."
+
+- *MGR*
+  * ceph-mgr supports a default dashboard
+  * ceph-mgr introduces a new pecan based rest API
+
+- *RGW*:
+
+  * RGW introduces server side encryption of uploaded objects with 3 options for
+    the management of encryption keys, automatic encryption (only recommended for
+    test setups), customer provided keys similar to Amazon SSE-C specification and
+    using a key management service (Openstack Barbician) similar to Amazon SSE-KMS
+    specification.
+  * RGW's metadata search with ElasticSearch now supports end user requests
+    serviced via RGW itself and now supports custom metadata fields
+  * RGW has consolidated the several metadata index pools via the use of rados
+    namespaces
+  * RGW now supports dynamic bucket index sharding
+
```

Comment:
Please add something like:

* RGW now has preliminary support for S3 style bucket policies.